.github/workflows/ci.yml

```diff
@@ -1,27 +1,14 @@
 name: CI
-on:
+"on":
```

Comment:
Weird that this needs to be in quotes when the other keys don't

Reply:
Is it a keyword or something?

Reply:
not that I've been able to find :thinking: